### DISCUSSION TITLE
Revisit limits in the protocol

### DISCUSSION THREAD

**Discussion by lunfardo314** [iotaledger/tips#61] [Ideas]
1. It seems that once we have a fixed limit on the size of transaction, other transaction level limits, such as maximum number of outputs, inputs and number of different native toke  IDs are artificial and useless
2. What is missing, it is limits on the output. The maximum number of native token IDs is discussed [here](https://github.com/iotaledger/tips/discussions/60#discussion-3843121)

**Comment by Wollac** [comment]
1. I don't think that those limits are useless at all, for my reasoning see https://github.com/iotaledger/tips/discussions/60#discussioncomment-2100428
2. I don't understand this point. Do you mean that the only relevant limit is that the numbers of tokens per output must not be larger than the number of outputs allowed in a transaction? In that case I would also refer to https://github.com/iotaledger/tips/discussions/60#discussioncomment-2100428

**Reply by lunfardo314** [reply]
I put some explanation [in another thread](https://github.com/iotaledger/tips/discussions/60#discussioncomment-2100428)
